InvoiceMaker/InvoiceMaker/Models/WorkDone.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.Models
+{
+    public class WorkDone
+    {
+
+        //private Client _client;
+        //private WorkType _workType;
+
+         public Client Client
+        {
+            get;set;
+        }
+        public WorkType WorkType
+        {
+            get;set;
+        }
+        
+        public WorkDone()
+        {
+
+        }
+        [Required,Column("ClientId")]
+        public int ClientId
+        {
+            get
+            {
+                return Client.Id;
+
+            }
+        }
+
+        [Column("WorkTypeRate")]
+        public decimal WorkTypeRate
+        {
+            get
+            {
+                return WorkType.Rate;
+            }
+        }
+        [Required, Column("WorkTypeId")]
+        public int WorkTypeId
+        {
+            get
+            {
+                return WorkType.Id;
+
+            }
+        }
+        [Column("StartedOn")]
+        public DateTimeOffset StartedOn
+        {
+            get;
+            private set;
+        }
+
+        [Column("EndedOn")]
+        public DateTimeOffset? EndedOn
+        {
+            get; private set;
+        }
+        [Column("ClientName")]
+        public string ClientName
+        {
+            get
+            {
+                return Client.Name;
+            }
+        }
+        [Column("WorkTypeName")]
+        public string WorkTypeName
+        {
+            get
+            {
+                return WorkType.Name;
+            }
+        }
+        
+        public int Id
+        {
+            get; private set;
```

Comment:
Remove the `private` access modifier from this setter. Making a property setter private when working with EF will slow down retrieving data from the database.